src/core/evaluator.js

```diff
@@ -1130,7 +1133,8 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
               var offset;
               for (var j = 0, jj = items.length; j < jj; j++) {
                 if (typeof items[j] === 'string') {
-                  buildTextGeometry(items[j], textChunk);
+                  buildTextGeometry(items[j], textChunk,
+                                   (j > 0 ? items[j - 1] : 0));
```

Comment:
How about passing in `null` instead when `j === 0`, i.e  changing it to
`(j > 0 ? items[j - 1] : null)`? That way, you'll avoid a pointless division at [line 958](https://github.com/mozilla/pdf.js/pull/6019/files#diff-0b94c2e77a5259f7a728122fdbf9f46aR958).

Also, can you please add *one* more space to the indentation of this line.